addons/resource/migrations/12.0.1.1/post-migration.py

```diff
@@ -47,13 +47,24 @@ def fill_resource_calendar_hours_per_day(env):
         calendar._onchange_hours_per_day()
 
 
+def fill_resource_test_resource_calendar_id(cr):
```

Comment:
Watch out: This a test only model! It shouldn't exist in regular databases

Reply:
I don't know why, but the table of this `ResourceTest(models.Model)` model is created :S

Reply:
Yes, on demo databases, but what happen on real ones? Any way, you shouldn't care about it although created.

Reply:
Ok, but it's rare that the openupgrade analysis detected this, because I think it doesn't use demo data.